defaults/main.yml

```diff
@@ -0,0 +1,61 @@
+redis_unixsocketperm: 700
+redis_syslog_enabled: "no"
+redis_syslog_ident: "redis"
+redis_syslog_facility: "local0"
+redis_repl_ping_slave_period: 10
+redis_repl_timeout: 60
+redis_repl_backlog_size: "1mb"
+redis_repl_backlog_ttl: 3600
+redis_bind: 127.0.0.1
+redis_protected_mode: yes
+redis_port: 6379
+redis_tcp_backlog: 511
+redis_timeout: 0
+redis_tcp_keepalive: 300
+redis_daemonize: no
+redis_supervised: no
+redis_pidfile: /var/run/redis_6379.pid
```

Comment:
What is this 6379?

Reply:
default right out of the redis conf

Reply:
But will this also work when I update redis?

Reply:
why should it not work its a socket path redis does not read shit from it

Reply:
I can change it to /var/run/redis.pid

Reply:
Please don't make me responsible when our redis breaks